src/twisted/protocols/test/test_tls.py

```diff
@@ -42,7 +44,8 @@
 
 from twisted.internet.error import ConnectionDone, ConnectionLost
 from twisted.internet.defer import Deferred, gatherResults
-from twisted.internet.protocol import Protocol, ClientFactory, ServerFactory
+from twisted.internet.protocol import (Protocol, ClientFactory, ServerFactory,
```

Comment:
this is fine, but to reduce future dif size, I would prefer to have this list alphabetically ordered and each import on a line.


```
from twisted.internet.protocol import (
    ClientServerFactory,
    Factory,
    Protocol,
    ServerFactory,
    )
```